scheduler/task.go

```diff
@@ -239,6 +239,7 @@ func (t *task) Enable() error {
 		return ErrTaskNotDisabled
 	}
 	t.state = core.TaskStopped
+	t.lastFireTime = time.Time{}
```

Comment:
This line needs to be removed before merge. PR #808 resolves the issue with the missed column being incremented.